nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/README.md

```diff
@@ -1,11 +1,31 @@
-# {{ cookiecutter.pipeline_name }} Documentation
+# {{ cookiecutter.pipeline_name }}
+
+{{ cookiecutter.pipeline_short_description }}
+
+[![Build Status](https://travis-ci.org/{{ cookiecutter.github_repo }}.svg?branch=master)](https://travis-ci.org/{{ cookiecutter.github_repo }})
```

Comment:
These readme badges go on the main `README.md` file, this one is `docs/README.md`. I don't think we need them in both (harder to keep the version numbers up to date), so please remove this guys :)